packages/config/config/devices/0x0384/ha-zw-5pa.json

```diff
@@ -55,104 +53,103 @@
 		},
 		{
 			"#": "101",
-			"label": "Setting Time For Sending Meter Report (watt)",
-			"description": "The interval time of sending meter report (watt) to associated device (group lifeline)",
+			"label": "Time For Sending Meter Report (watt)",
 			"valueSize": 4,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 2678400,
 			"defaultValue": 600,
 			"unsigned": true,
 			"options": [
 				{
-					"label": "The function is disabled",
+					"label": "Disabled",
 					"value": 0
 				}
 			]
 		},
 		{
 			"#": "102",
-			"label": "Setting Time For Sending Meter Report (kwh)",
-			"description": "The interval of sending meter report (kwh) to association device (group lifeline)",
+			"label": "Time For Sending Meter Report (kwh)",
 			"valueSize": 4,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 2678400,
 			"defaultValue": 3600,
 			"unsigned": true,
 			"options": [
 				{
-					"label": "The function is disabled",
+					"label": "Disabled",
 					"value": 0
 				}
 			]
 		},
 		{
 			"#": "103",
-			"label": "Setting Time For Sending Meter Report (voltage)",
-			"description": "The interval of sending meter report (voltage) to association device (group lifeline)",
+			"label": "Time For Sending Meter Report (voltage)",
 			"valueSize": 4,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 2678400,
 			"defaultValue": 0,
 			"unsigned": true,
 			"options": [
 				{
-					"label": "The function is disabled",
+					"label": "Disabled",
 					"value": 0
 				}
 			]
 		},
 		{
 			"#": "104",
-			"label": "Setting Time For Sending Meter Report (current)",
-			"description": "The interval time of sending meter report (current) to associated device (group lifeline)",
+			"label": "Time For Sending Meter Report (current)",
 			"valueSize": 4,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 2678400,
 			"defaultValue": 0,
 			"unsigned": true,
 			"options": [
 				{
-					"label": "The function is disabled",
+					"label": "Disabled",
 					"value": 0
 				}
 			]
 		},
 		{
 			"#": "16",
-			"label": "The Value Here Represents Minimum Change in Wattage (in Terms of Wattage) For a Meter Report to Be Sent",
-			"description": "When the change value of load power exceeds the setting value, smart plug will send meter report (watt) to associated device (group lifeline)",
+			"label": "Minimum Change in Wattage For a Meter Report",
 			"valueSize": 2,
-			"minValue": -32768,
+			"unit": "W",
+			"minValue": 0,
 			"maxValue": 32767,
```

Comment:
Are you sure about this limit? If -32768 was interpreted as unsigned before, this limit could also have been 0...65535. Please double-check if possible.

Reply:
Ok, yeah that is because some hubs don't have the option to enter the value as unsigned. I'll change it.